src/ApiSummary.js

```diff
@@ -397,7 +397,11 @@ export class ApiSummary extends AmfHelperMixin(LitElement) {
   _serverListTemplate(server) {
     const { baseUri, protocols } = this;
     const uri = this._computeBaseUri(server, baseUri, protocols);
-    return html`<li>${uri}</li>`;
+    const description = this._computeDescription(server);
+    return html`<li>
+      ${uri}
+      <div class="server-description">${description}</div>
```

Comment:
Can this be a markdown? If so it should also be wrapped with `arc-markdown` element.

Reply:
Yep, now using arc-marked for server description